src/NServiceBus.Core/Licensing/LogErrorOnInvalidLicenseBehavior.cs

```diff
@@ -0,0 +1,19 @@
+namespace NServiceBus
+{
+    using System;
+    using System.Threading.Tasks;
+    using Logging;
+    using Pipeline;
+
+    class LogErrorOnInvalidLicenseBehavior : Behavior<PhysicalMessageProcessingContext>
```

Comment:
How about I put a if(IsDebuggerAttached) in the acceptance tests so we can check it with a manual test run in debug mode?

Would that be enough?